src/NumberInput/NumberInput.tsx

```diff
@@ -10,12 +10,12 @@ import styled from 'styled-components'
 import { Box } from '../Box'
 import { Text } from '../Text'
 import { Icon } from '../Icon'
-import { Field, CommonFieldTypes } from '../Field'
+import { Field, CommonFieldProps } from '../fields/Field'
 
 import { theme } from '../theme'
 import { useUniqueId } from '../utils/id'
 
-export interface Props extends CommonFieldTypes {
+export interface Props extends CommonFieldProps {
   type?: 'number' | 'tel'
```

Comment:
Not sure `'number' and 'tel'` are very clear, could we change to something more helpful?

Reply:
This is part of the API for `NumberInput` so changing might be breaking some stuff
Better keep it as it is for now but might be worth to have a look later on 👍